### PR TITLE
refactor(frontend): hide partially the footer DFINITY string [OISY-62]

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

The idea is to hide the "Developed with..." string for middle screens, leaving just the DFINITY logo. However, for narrower middle screen, hide the logo too.

# Changes

- Make the entire string be an hyperlink.
- Modify the footer so that it is hidden on narrow middle screens, and only a logo on middle screens.
- The rest is unchanged.

# Tests

https://github.com/user-attachments/assets/72d9e1b8-65c0-41f8-a0fc-c75f8886d0b5

### CODE REVIEW

File: src/frontend/src/lib/components/core/Footer.svelte [hunk 1]
```diff
@@ -34,20 +37,24 @@
...
-				href="https://dfinity.org"
-				ariaLabel={replaceOisyPlaceholders($i18n.footer.alt.dfinity)}
-				color="blue"
-				iconVisible={false}
-			>
-				{replaceOisyPlaceholders($i18n.footer.text.dfinity)}
-			</ExternalLink>
-		</span>
+		<ExternalLink
+			href="https://dfinity.org"
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/core/Footer.svelte:48]: Wait what, I missed that in a previous review. It's a link to DFINITY, not the Internet Computer?!!??!? That's correct and expected?

File: src/frontend/src/lib/components/core/Footer.svelte [hunk 2]
```diff
@@ -34,20 +37,24 @@
...
+			ariaLabel={replaceOisyPlaceholders($i18n.footer.alt.dfinity)}
+			iconVisible={false}
+		>
+			<div class="flex flex-row items-center gap-2">
+				<IconDfinity />
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/core/Footer.svelte:53]: I missed that too, did I review the previous PR 😅. It's `IconDfinity` not `IconInternetComputer` or something?

**Review by peterpeterparker — APPROVED** [review]
LGTM, thanks

Two questions unrelated to this PR, therefore approving the code.

File: src/frontend/src/lib/components/core/Footer.svelte (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/core/Footer.svelte:48]: I am pretty sure... I will double confirm, but the string says `Developed with... by DFINITY`, so I would imagine that is where it should point. But good question, I will ask

File: src/frontend/src/lib/components/core/Footer.svelte (hunk 2 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/core/Footer.svelte:53]: eheh same as above... i will re-check

File: src/frontend/src/lib/components/core/Footer.svelte (hunk 1 above)
Comment (peterpeterparker) [reply src/frontend/src/lib/components/core/Footer.svelte:48]: Please do. Thank in advance.

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/core/Footer.svelte:48]: Just as follow up: I checked with both PM and Design team and that is indeed on purpose all DFINITY related.

File: src/frontend/src/lib/components/core/Footer.svelte (hunk 2 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/core/Footer.svelte:53]: Just as follow up: I checked with both PM and Design team and that is indeed on purpose all DFINITY related.

File: src/frontend/src/lib/components/core/Footer.svelte (hunk 1 above)
Comment (peterpeterparker) [reply src/frontend/src/lib/components/core/Footer.svelte:48]: Thanks for double checking.

**Pull request merged**